### ISSUE TITLE
Minecraft crashes in EndCaveBiome

### ISSUE THREAD

**Issue by Reniel80** [quiqueck/BetterEnd#275]
### What happened?

EndCaveBiome is in unnamed module of loader

### BetterEnd

4.0.8

### BCLib

3.0.11

### Fabric API

0.86.0

### Fabric Loader

0.14.21

### Minecraft

1.20.1

### Relevant log output

```shell
This is the relevant info:

Caused by: java.lang.ClassCastException: class org.betterx.bclib.api.v2.levelgen.biomes.BCLBiome cannot be cast to class org.betterx.betterend.world.biome.cave.EndCaveBiome (org.betterx.bclib.api.v2.levelgen.biomes.BCLBiome and org.betterx.betterend.world.biome.cave.EndCaveBiome are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @3ffc5af1)

---------------------------------------------------------------------------------------

This is the full crash report:

---- Minecraft Crash Report ----
// I let you down. Sorry :(

Time: 2023-07-21 23:15:51
Description: Watching Server

java.lang.Error: Watchdog
	at net.minecraft.class_3178.run(class_3178.java:50)
	at java.base/java.lang.Thread.run(Thread.java:833)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server Watchdog
Stacktrace:
	at net.minecraft.class_3178.run(class_3178.java:50)

-- Thread Dump --
Details:
	Threads: "Reference Handler" daemon prio=10 Id=2 RUNNABLE
	at java.base@18.0.2.1/java.lang.ref.Reference.waitForReferencePendingList(Native Method)
	at java.base@18.0.2.1/java.lang.ref.Reference.processPendingReferences(Reference.java:253)
	at java.base@18.0.2.1/java.lang.ref.Reference$ReferenceHandler.run(Reference.java:215)


"Finalizer" daemon prio=8 Id=3 WAITING on java.lang.ref.ReferenceQueue$Lock@467cffd2
	at java.base@18.0.2.1/java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@467cffd2
	at java.base@18.0.2.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
	at java.base@18.0.2.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
	at java.base@18.0.2.1/java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:183)


"Signal Dispatcher" daemon prio=9 Id=4 RUNNABLE


"Attach Listener" daemon prio=5 Id=5 RUNNABLE


"Notification Thread" daemon prio=9 Id=13 RUNNABLE


"Common-Cleaner" daemon prio=8 Id=14 TIMED_WAITING on java.lang.ref.ReferenceQueue$Lock@48bb096d
	at java.base@18.0.2.1/java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@48bb096d
	at java.base@18.0.2.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
	at java.base@18.0.2.1/jdk.internal.ref.CleanerImpl.run(CleanerImpl.java:140)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)
	at java.base@18.0.2.1/jdk.internal.misc.InnocuousThread.run(InnocuousThread.java:162)


"JNA Cleaner" daemon prio=5 Id=29 WAITING on java.lang.ref.ReferenceQueue$Lock@4c8494a2
	at java.base@18.0.2.1/java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@4c8494a2
	at java.base@18.0.2.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
	at java.base@18.0.2.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
	at com.sun.jna.internal.Cleaner$1.run(Cleaner.java:58)


"Java2D Disposer" daemon prio=10 Id=31 WAITING on java.lang.ref.ReferenceQueue$Lock@9860356
	at java.base@18.0.2.1/java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@9860356
	at java.base@18.0.2.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
	at java.base@18.0.2.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
	at java.desktop@18.0.2.1/sun.java2d.Disposer.run(Disposer.java:145)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"FileSystemWatchService" daemon prio=5 Id=32 RUNNABLE (in native)
	at java.base@18.0.2.1/sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at java.base@18.0.2.1/sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:1065)
	at java.base@18.0.2.1/sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:587)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"Config Lib config file watcher" daemon prio=5 Id=33 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@28a5ffdb
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@28a5ffdb
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3464)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3435)
	at java.base@18.0.2.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.LinkedBlockingDeque.takeFirst(LinkedBlockingDeque.java:485)
	at java.base@18.0.2.1/java.util.concurrent.LinkedBlockingDeque.take(LinkedBlockingDeque.java:673)
	...


"Thread-2" daemon prio=5 Id=34 TIMED_WAITING
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:376)
	at com.electronwill.nightconfig.core.file.FileWatcher$WatcherThread.run(FileWatcher.java:190)


"FileSystemWatchService" daemon prio=5 Id=35 RUNNABLE (in native)
	at java.base@18.0.2.1/sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at java.base@18.0.2.1/sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:1065)
	at java.base@18.0.2.1/sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:587)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"JsonConfig-Worker-1" daemon prio=5 Id=40 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@38456bc3
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@38456bc3
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3464)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3435)
	at java.base@18.0.2.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
	...


"JsonConfig-Worker-2" daemon prio=5 Id=41 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@508cc9ed
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@508cc9ed
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3464)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3435)
	at java.base@18.0.2.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.LinkedBlockingDeque.takeFirst(LinkedBlockingDeque.java:485)
	at java.base@18.0.2.1/java.util.concurrent.LinkedBlockingDeque.take(LinkedBlockingDeque.java:673)
	...

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@73b693c0


"FileSystemWatchService" daemon prio=5 Id=42 RUNNABLE (in native)
	at java.base@18.0.2.1/sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at java.base@18.0.2.1/sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:1065)
	at java.base@18.0.2.1/sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:587)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"ForkJoinPool.commonPool-worker-4" daemon prio=5 Id=48 WAITING on java.util.concurrent.ForkJoinPool@69a9fc19
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@69a9fc19
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1724)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"ForkJoinPool.commonPool-worker-5" daemon prio=5 Id=50 WAITING on java.util.concurrent.ForkJoinPool@69a9fc19
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@69a9fc19
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1724)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"ForkJoinPool.commonPool-worker-6" daemon prio=5 Id=51 WAITING on java.util.concurrent.ForkJoinPool@69a9fc19
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@69a9fc19
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1724)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"ForkJoinPool.commonPool-worker-7" daemon prio=5 Id=52 TIMED_WAITING on java.util.concurrent.ForkJoinPool@69a9fc19
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@69a9fc19
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkUntil(LockSupport.java:410)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1726)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"Worker-Main-1" daemon prio=5 Id=53 WAITING on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1724)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"Timer hack thread" daemon prio=5 Id=55 TIMED_WAITING
	at java.base@18.0.2.1/java.lang.Thread.sleep(Native Method)
	at net.minecraft.class_156$9.run(class_156.java:839)


"Yggdrasil Key Fetcher" daemon prio=5 Id=56 TIMED_WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7dfde8be
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7dfde8be
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
	at java.base@18.0.2.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
	at java.base@18.0.2.1/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
	at java.base@18.0.2.1/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	...


"Worker-Main-2" daemon prio=5 Id=57 WAITING on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1724)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"Worker-Main-3" daemon prio=5 Id=58 WAITING on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1724)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"Worker-Main-4" daemon prio=5 Id=59 WAITING on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1724)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"Worker-Main-5" daemon prio=5 Id=60 TIMED_WAITING on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkUntil(LockSupport.java:410)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1726)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"Worker-Main-6" daemon prio=5 Id=61 WAITING on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1724)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"Worker-Main-7" daemon prio=5 Id=62 WAITING on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.ForkJoinPool@6ef58dd0
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1724)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
	at java.base@18.0.2.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)


"DestroyJavaVM" prio=5 Id=65 RUNNABLE


"FileSystemWatchService" daemon prio=8 Id=66 RUNNABLE (in native)
	at java.base@18.0.2.1/sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at java.base@18.0.2.1/sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:1065)
	at java.base@18.0.2.1/sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:587)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"Server console handler" daemon prio=8 Id=68 RUNNABLE (in native)
	at java.base@18.0.2.1/java.io.FileInputStream.readBytes(Native Method)
	at java.base@18.0.2.1/java.io.FileInputStream.read(FileInputStream.java:276)
	at java.base@18.0.2.1/java.io.BufferedInputStream.read1(BufferedInputStream.java:282)
	at java.base@18.0.2.1/java.io.BufferedInputStream.read(BufferedInputStream.java:343)
	-  locked java.io.BufferedInputStream@4b88854d
	at java.base@18.0.2.1/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:270)
	at java.base@18.0.2.1/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:313)
	at java.base@18.0.2.1/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:188)
	-  locked java.io.InputStreamReader@6de7e111
	at java.base@18.0.2.1/java.io.InputStreamReader.read(InputStreamReader.java:176)
	...


"Netty Server IO #0" daemon prio=5 Id=69 RUNNABLE (in native)
	at java.base@18.0.2.1/sun.nio.ch.WEPoll.wait(Native Method)
	at java.base@18.0.2.1/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
	at java.base@18.0.2.1/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
	-  locked io.netty.channel.nio.SelectedSelectionKeySet@2fbe6a70
	-  locked sun.nio.ch.WEPollSelectorImpl@3d94cf21
	at java.base@18.0.2.1/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
	at io.netty.channel.nio.SelectedSelectionKeySetSelector.select(SelectedSelectionKeySetSelector.java:68)
	at io.netty.channel.nio.NioEventLoop.select(NioEventLoop.java:879)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:526)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	...


"Server Watchdog" daemon prio=8 Id=73 RUNNABLE
	at java.management@18.0.2.1/sun.management.ThreadImpl.dumpThreads0(Native Method)
	at java.management@18.0.2.1/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:521)
	at java.management@18.0.2.1/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:509)
	at net.minecraft.class_3178.run(class_3178.java:47)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"Thread-12" daemon prio=8 Id=74 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@32b99733
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@32b99733
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"Thread-13" daemon prio=8 Id=75 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@32b99733
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@32b99733
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"Thread-14" daemon prio=8 Id=76 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@32b99733
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@32b99733
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"IO-Worker-12" prio=8 Id=81 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@6084171c
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@6084171c
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"Netty Server IO #1" daemon prio=5 Id=82 RUNNABLE (in native)
	at java.base@18.0.2.1/sun.nio.ch.WEPoll.wait(Native Method)
	at java.base@18.0.2.1/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
	at java.base@18.0.2.1/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
	-  locked io.netty.channel.nio.SelectedSelectionKeySet@3cb50c96
	-  locked sun.nio.ch.WEPollSelectorImpl@274321e6
	at java.base@18.0.2.1/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
	at io.netty.channel.nio.SelectedSelectionKeySetSelector.select(SelectedSelectionKeySetSelector.java:68)
	at io.netty.channel.nio.NioEventLoop.select(NioEventLoop.java:879)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:526)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	...


"Netty Server IO #2" daemon prio=5 Id=83 RUNNABLE (in native)
	at java.base@18.0.2.1/sun.nio.ch.WEPoll.wait(Native Method)
	at java.base@18.0.2.1/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
	at java.base@18.0.2.1/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
	-  locked io.netty.channel.nio.SelectedSelectionKeySet@59bd2a32
	-  locked sun.nio.ch.WEPollSelectorImpl@1941faee
	at java.base@18.0.2.1/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
	at io.netty.channel.nio.SelectedSelectionKeySetSelector.select(SelectedSelectionKeySetSelector.java:68)
	at io.netty.channel.nio.NioEventLoop.select(NioEventLoop.java:879)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:526)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	...


"Netty Server IO #3" daemon prio=5 Id=84 RUNNABLE (in native)
	at java.base@18.0.2.1/sun.nio.ch.WEPoll.wait(Native Method)
	at java.base@18.0.2.1/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
	at java.base@18.0.2.1/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
	-  locked io.netty.channel.nio.SelectedSelectionKeySet@d0494ec
	-  locked sun.nio.ch.WEPollSelectorImpl@3376d8bb
	at java.base@18.0.2.1/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
	at io.netty.channel.nio.SelectedSelectionKeySetSelector.select(SelectedSelectionKeySetSelector.java:68)
	at io.netty.channel.nio.NioEventLoop.select(NioEventLoop.java:879)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:526)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	...


"IO-Worker-13" prio=8 Id=85 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@6084171c
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@6084171c
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)


"IO-Worker-15" prio=8 Id=87 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@6084171c
	at java.base@18.0.2.1/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@6084171c
	at java.base@18.0.2.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
	at java.base@18.0.2.1/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
	at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)



Stacktrace:
	at net.minecraft.class_3178.run(class_3178.java:50)
	at java.base/java.lang.Thread.run(Thread.java:833)

-- Performance stats --
Details:
	Random tick rate: 3
	Level stats: ResourceKey[minecraft:dimension / minecraft:overworld]: players: 0, entities: 112,33,72,565,23,0,542 [minecraft:sheep:8,minecraft:pig:6,minecraft:chicken:4,myloot:loot_chest_minecart:3,minecraft:item:3], block_entities: 76 [minecraft:mob_spawner:27,minecraft:campfire:10,ad_astra:cable:6,betternether:forge:4,minecraft:beehive:3], block_ticks: 19, fluid_ticks: 161, chunk_source: Chunks[S] W: 3109 E: 112,33,72,565,23,0,542,
ResourceKey[minecraft:dimension / ad_astra:mars_orbit]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / immersive_portals:alternate3]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / immersive_portals:alternate2]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / minecraft:the_nether]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / ad_astra:mercury]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / ad_astra:glacio_orbit]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / ad_astra:venus_orbit]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / immersive_portals:alternate4]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / ad_astra:mars]: players: 0, entities: 28,17,19,81,16,0,65 [minecraft:item:3,mca:female_villager:2,mythicmobs:drake:2,mythicmobs:automaton:2,friendsandfoes:glare:1], block_entities: 181 [ad_astra:cable:114,ad_astra:coal_generator:28,ad_astra:solar_panel:11,ad_astra:sliding_door:10,ad_astra:oxygen_distributor:7], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 1089 E: 28,17,19,81,16,0,65,
ResourceKey[minecraft:dimension / immersive_portals:alternate1]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / ad_astra:mercury_orbit]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / ad_astra:moon_orbit]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / ad_astra:earth_orbit]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / minecraft:the_end]: players: 1, entities: 111,1,49,602,12,0,601 [minecraft:player:1], block_entities: 2 [minecraft:brewing_stand:2], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 2399 E: 111,1,49,602,12,0,601,
ResourceKey[minecraft:dimension / ad_astra:venus]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / immersive_portals:alternate5]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / ad_astra:glacio]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / ad_astra:moon]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0
Stacktrace:
	at net.minecraft.class_3178.run(class_3178.java:50)
	at java.base/java.lang.Thread.run(Thread.java:833)

-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 18.0.2.1, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
	Memory: 969693984 bytes (924 MiB) / 4127195136 bytes (3936 MiB) up to 8589934592 bytes (8192 MiB)
	CPUs: 8
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
	Identifier: Intel64 Family 6 Model 158 Stepping 9
	Microarchitecture: Kaby Lake
	Frequency (GHz): 2.81
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: NVIDIA GeForce GTX 1060
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 3072.00
	Graphics card #0 deviceId: 0x1c60
	Graphics card #0 versionInfo: DriverVersion=30.0.15.1165
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Virtual memory max (MB): 32211.96
	Virtual memory used (MB): 9707.03
	Swap memory total (MB): 15872.00
	Swap memory used (MB): 136.55
	JVM Flags: 1 total; -Xmx8G
	Fabric Mods: 
		ad_astra: Ad Astra 1.15.2
		adorn: Adorn 5.0.0+1.20.1
		allthetrims: AllTheTrims 3.1.2
		appleskin: AppleSkin 2.5.0+mc1.20
		aquamirae: Aquamirae 6
		architectury: Architectury 9.1.10
		balm-fabric: Balm 7.0.4
		bclib: BCLib 3.0.11
			wunderlib: WunderLib 1.1.5
		bettercombat: Better Combat 1.7.2+1.20.1
		betterend: Better End 4.0.8
		betternether: Better Nether 9.0.8
		biomespawnpoint: Biome Spawn Point 2.0
		bookshelf: Bookshelf 20.0.1
		bosses_of_mass_destruction: Bosses of Mass Destruction (Beta) 1.7.1-1.20
			maelstrom_library: Maelstrom Library 1.6-1.20
			multipart_entities: MultipartEntities 1.5-1.20
		botarium: Botarium 2.1.2
			team_reborn_energy: Energy 2.2.0
		cardinal-components: Cardinal Components API 5.2.1
			cardinal-components-base: Cardinal Components API (base) 5.2.1
			cardinal-components-block: Cardinal Components API (blocks) 5.2.1
			cardinal-components-chunk: Cardinal Components API (chunks) 5.2.1
			cardinal-components-entity: Cardinal Components API (entities) 5.2.1
			cardinal-components-item: Cardinal Components API (items) 5.2.1
			cardinal-components-level: Cardinal Components API (world saves) 5.2.1
			cardinal-components-scoreboard: Cardinal Components API (scoreboard) 5.2.1
			cardinal-components-world: Cardinal Components API (worlds) 5.2.1
		carryon: Carry On 2.1.1.2
		chunkloaders: Chunk Loaders 1.2.5
		chunky: Chunky 1.3.92
		cloth-config: Cloth Config v11 11.1.106
			cloth-basic-math: cloth-basic-math 0.6.1
		clumps: Clumps 12.0.0.3
		collective: Collective 6.65
		comforts: Comforts 6.3.3+1.20.1
		companion: Companion 5.0.2
		completeconfig: CompleteConfig 2.4.0
			completeconfig-base: completeconfig-base 2.4.0
		connectivity: Connectivity Mod 1.20.1-4.4
		corgilib: CorgiLib 4.0.0.0
			com_electronwill_night-config_core: core 3.6.6
			com_electronwill_night-config_toml: toml 3.6.6
		creativecore: CreativeCore 2.10.28
			net_minecraftforge_eventbus: eventbus 6.0.3
		cristellib: Cristel Lib 1.1.0
		ctov: ChoiceTheorem's Overhauled Village 3.3.2
		dehydration: Dehydration 1.3.4
		doubledoors: Double Doors 5.0
		dragondropselytra: Dragon Drops Elytra 3.0
		easyanvils: Easy Anvils 8.0.0
		easymagic: Easy Magic 8.0.0
		elytraslot: Elytra Slot 6.3.0+1.20.1
		emi: EMI 1.0.11+1.20.1+fabric
		emi_loot: EMI Loot 0.6.4.1+1.20.1
		enchantinginfuser: Enchanting Infuser 8.0.1
		enchantment-transfer: Enchantment Transfer 0.0.6
		explorify: Explorify v1.3.0-mc1.20u1.20.1
		fabric-api: Fabric API 0.86.0+1.20.1
			fabric-api-base: Fabric API Base 0.4.30+7abfd51577
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.35+4d8536c977
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.10+b3afc78b77
			fabric-block-api-v1: Fabric Block API (v1) 1.0.9+e022e5d177
			fabric-command-api-v1: Fabric Command API (v1) 1.2.33+f71b366f77
			fabric-command-api-v2: Fabric Command API (v2) 2.2.12+b3afc78b77
			fabric-commands-v0: Fabric Commands (v0) 0.2.50+df3654b377
			fabric-containers-v0: Fabric Containers (v0) 0.1.62+df3654b377
			fabric-content-registries-v0: Fabric Content Registries (v0) 4.0.9+b3afc78b77
			fabric-convention-tags-v1: Fabric Convention Tags 1.5.4+a1a980da77
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.18+aeb40ebe77
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 12.2.0+6c31357e77
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.52+b3afc78b77
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.22+b3afc78b77
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.6.1+e91849a877
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.62+df3654b377
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.38+b04edc7a77
			fabric-item-api-v1: Fabric Item API (v1) 2.1.27+b3afc78b77
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.0.9+132c48c177
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.21+b3afc78b77
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.39+b3afc78b77
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.43+9e7660c677
			fabric-message-api-v1: Fabric Message API (v1) 5.1.7+3265161977
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.49+b3afc78b77
			fabric-networking-api-v1: Fabric Networking API (v1) 1.3.9+b3afc78b77
			fabric-networking-v0: Fabric Networking (v0) 0.3.49+df3654b377
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 11.1.1+6beca84877
			fabric-particles-v1: Fabric Particles (v1) 1.1.1+201a23a077
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.19+b3afc78b77
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.3.0+2e061fd477
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.34+b3afc78b77
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.27+b3afc78b77
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.5+ea08f9d877
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.11.9+132c48c177
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.28+b3afc78b77
			fabric-transfer-api-v1: Fabric Transfer API (v1) 3.3.0+cdf060b277
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 4.3.0+6c31357e77
		fabric-language-kotlin: Fabric Language Kotlin 1.10.8+kotlin.1.9.0
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.9.0
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.9.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.9.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.9.0
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.21.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.7.2
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.7.2
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.5.1
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.5.1
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.5.1
		fabricloader: Fabric Loader 0.14.21
		farmersdelight: Farmer's Delight 1.20.1-1.4.1
		forgeconfigapiport: Forge Config API Port 8.0.0
		forgottengraves: Forgotten Graves 1.20.1-3.2.0
		friendlyfire: FriendlyFire 18.0.2
		friendsandfoes: Friends&Foes 1.9.3
		fwaystones: Fabric Waystones 3.1.2+mc1.20
		geckolib: Geckolib 4.2.1
			com_eliotlash_mclib_mclib: mclib 20
		gravelminer: GravelMiner 16.0.2
		graveyard: The Graveyard 2.9.1
		hardcorerevival: Hardcore Revival 12.0.0
		healingcampfire: Healing Campfire 5.1
		iceberg: Iceberg 1.1.11
		immersive_aircraft: Immersive Aircraft 0.5.1+1.20.1
		immersive_portals: Immersive Portals 3.1.2
			com_github_llamalad7_mixinextras: MixinExtras 0.2.0-beta.9
			dimension_data_fix: Dimension Data Fix 2.0.0
			imm_ptl_core: Immersive Portals Core 3.1.2
			q_misc_util: QMiscUtil 3.1.2
		imst: Immersive structures 2.1.0
		infinitetrading: Infinite Trading 4.0
		inventorytotem: Inventory Totem 3.0
		jade: Jade 11.3.0
		jamlib: JamLib 0.6.0+1.20
		java: Java HotSpot(TM) 64-Bit Server VM 18
		journeymap: Journeymap 5.9.12
			journeymap-api-fabric: JourneyMap API 1.20-1.9-fabric-SNAPSHOT
		jumpoverfences: Jump Over Fences 1.3.0
		kiwi: Kiwi Library 11.0.0
		libraryferret: Library ferret 4.0.0
		logprot: Logprot Mod 1.20.1-3.0
		mca: Minecraft Comes Alive 7.5.3+1.20.1
		midnightlib: MidnightLib 1.4.1
		minecraft: Minecraft 1.20.1
		mineralchance: Mineral Chance 3.2
		moonlight: Moonlight 1.20-2.7.2
		morebows: More Bows Restrung 1.0.11+1.20.x
		mr_dungeons_andtaverns: Dungeons and Taverns 1-v2.1.4
		mru: Mineblock's Repeated Utilities 0.2.1+1.20
		mvs: Moog's Voyager Structures 4.0-1.20-fabric
		myloot: myLoot 4.0.2-1.20.1
			omega-config: OmegaConfig 1.2.3-1.18.1
		mythicmobs: Mythic Mobs 1.3.0
		nohostilesaroundcampfire: No Hostiles Around Campfire 5.6
		obscure_api: Obscure API 16
		origins: Origins 1.10.0
			apoli: Apoli 2.9.0
				additionalentityattributes: Additional Entity Attributes 1.3.0+1.20.0
				calio: Calio 1.11.0
				playerabilitylib: Pal 1.8.0
			reach-entity-attributes: Reach Entity Attributes 2.4.0
		origins-plus-plus: Origins++ 2.1.1
		owo: oωo 0.11.1+1.20
			blue_endless_jankson: jankson 1.2.2
		passiveshield: Passive Shield 3.2
		patchouli: Patchouli 1.20.1-81-FABRIC
			fiber: fiber 0.23.0-2
		pehkui: Pehkui 3.7.6+1.14.4-1.20.1
			kanos_config: Kanos Config 0.4.1+1.14.4-1.19.4
		philipsruins: Philip`s Ruins 1.20.1
		pickupnotifier: Pick Up Notifier 8.0.0
		player-animator: Player Animator 1.0.2-rc1+1.20
		polymorph: Polymorph 0.49.0+1.20.1
			spectrelib: SpectreLib 0.13.13+1.20.1
		puffish_skills: Pufferfish's Skills 0.5.0
		puzzleslib: Puzzles Lib 8.0.7
		quartzelv: Quartz Elevator 2.2.5+1.20
		resourcefulconfig: Resourcefulconfig 2.0.0
		resourcefullib: Resourceful Lib 2.1.4
			com_teamresourceful_yabn: yabn 1.0.3
		rightclickharvest: Right Click Harvest 3.2.2+1.19.x-1.20.1-fabric
		simplyswords: Simply Swords 1.48.1-1.21
		spoornpacks: SpoornPacks 5.0-1.20.1
			com_fasterxml_jackson_core_jackson-annotations: jackson-annotations 2.13.1
			com_fasterxml_jackson_core_jackson-core: jackson-core 2.13.1
			com_fasterxml_jackson_core_jackson-databind: jackson-databind 2.13.1
		structory: Structory 1.3.2
		structory_towers: Structory: Towers 1.0.3
		supermartijn642configlib: SuperMartijn642's Config Lib 1.1.7
		supermartijn642corelib: SuperMartijn642's Core Lib 1.1.11
		supplementaries: Supplementaries 1.20-2.5.21
		theprinter: The Printer 1.4.2+1.20
			datacriteria: Data Criteria 1.0.1+1.20
		things: Things 0.3.1+1.20
			lavender: Lavender 0.1.0-pre.6+1.20
				lavender-md: lavender-md 0.1.0-pre.3+1.20
				lavender-md-owo-ui: lavender-md-owo-ui 0.1.0-pre.3+1.20
		timecontrol: TimeControl 1.4.1
		toms_storage: Tom's Simple Storage Mod 1.6.4
		trashslot: TrashSlot 15.0.0
		travelersbackpack: Traveler's Backpack 1.20.1-9.1.2
		treeharvester: Tree Harvester 8.1
		trinkets: Trinkets 3.7.0
		usefulbackpacks: Useful Backpacks 2.0.1.121
		uteamcore: U Team Core 5.1.3.267
		villagersplus: Villagers Plus 2.0
		visualworkbench: Visual Workbench 8.0.0
		voicechat: Simple Voice Chat 1.20.1-2.4.15
		walljump: Wall Jump 1.3.6
			org_aeonbits_owner_owner: owner 1.0.12
		weakerspiderwebs: Weaker Spiderwebs 3.2
		woof: Woof 4.0.2+1.20.1
	Server Running: false
	Player Count: 1 / 20; [class_3222['Hajime'/146, l='ServerWorld minecraft:the_end world', x=1675.90, y=62.55, z=754.86]]
	Data Packs: vanilla, Supplementaries Generated Pack, fabric, spoornpacks, theprinter:default (incompatible)
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Experimental
	Is Modded: Definitely; Server brand changed to 'fabric'
	Type: Dedicated Server (map_server.txt)
```

**Comment by quiqueck** [comment]
duplicate of #273.

**Comment by quiqueck** [comment]
The other thread contains a solution (for now) which was downgrading to FAPI 0.85.
